src/helpers/encodeParams.js

```diff
@@ -0,0 +1,30 @@
+const serializeParam = (prefix, param, value) => {
+  prefix = prefix ? `${prefix}[${param}]` : param;
```

Comment:
it's not common to over-write JS function params like this sadly.
think you have to re-name the variable